tests/test_cli.py

```diff
@@ -732,3 +733,27 @@ def test_CustomJsonEncoder_with_non_json_serialisable_object(self):
         encoder = CustomJsonEncoder()
         response = encoder.encode(datetime.datetime(2016, 5, 3))
         assert response == '"2016-05-03 00:00:00"'
+
+    def test_get_stack_or_env_with_stack(self):
+        ctx = MagicMock(obj={
+            "sceptre_dir": sentinel.sceptre_dir,
+            "user_variables": sentinel.user_variables
+        })
+        stack, env = get_stack_or_env(ctx, "stack.yaml")
+        self.mock_ConfigReader.assert_called_once_with(
+            sentinel.sceptre_dir, sentinel.user_variables
+        )
+        assert isinstance(stack, Stack)
+        assert env is None
+
+    def test_get_stack_or_env_with_env(self):
+        ctx = MagicMock(obj={
+            "sceptre_dir": sentinel.sceptre_dir,
+            "user_variables": sentinel.user_variables
+        })
+        stack, env = get_stack_or_env(ctx, "environment/dir")
+        self.mock_ConfigReader.assert_called_once_with(
+            sentinel.sceptre_dir, sentinel.user_variables
+        )
+        assert isinstance(env, Environment)
+        assert stack is None
```

Comment:
Could we add one more test to make sure `environment/dir/stack.yaml` returns a stack?